manifest/parse.go

```diff
@@ -40,8 +40,13 @@ func (m metadata) String() string {
 	if len(sp) > 1 {
 		apiBase = strings.Join(sp[:len(sp)-1], "/")
 	}
-
-	return fmt.Sprintf("%s, %s, %s (%s)", m.Metadata.Namespace, m.Metadata.Name, m.Kind, apiBase)
+	name := m.Metadata.Name
+	if a := m.Metadata.Annotations; a != nil {
+		if baseName, ok := a["helm.sh/base-name"]; ok {
```

Comment:
Can we instead use anything else that communicates better about the fact helm-diff is independent from the helm project?(whereas `helm.sh/` prefix makes it look like it's from the upstream project

```suggestion
		if baseName, ok := a["helm-diff-base-name"]; ok {
```

Reply:
How about "helm-diff/base-name" ?